javascript/rest-nuxtjs/README.md

```diff
@@ -0,0 +1,252 @@
+# Fullstack Example with NuxtJs (REST API)
+
+This example shows how to implement a **fullstack app with [NuxtJs](https://nuxtjs.org//)** using [Vue](https://vuejs.org/) (frontend), [Express](https://expressjs.com/) and [Prisma Client](https://www.prisma.io/docs/reference/tools-and-interfaces/prisma-client) (backend). It uses a SQLite database file with some initial dummy data which you can find at [`./prisma/dev.db`](./prisma/dev.db).
+
+## How to use
+
+### 1. Download example & install dependencies
+
+Download this example:
+
+```
+curl https://codeload.github.com/prisma/prisma-examples/tar.gz/latest | tar -xz --strip=2 prisma-examples-latest/javascript/rest-nuxtjs
+```
+
+Install npm dependencies:
+```
+cd rest-nuxtjs
+npm install
+```
+
+Note that this also generates Prisma Client JS into `node_modules/@prisma/client` via a `postinstall` hook of the `@prisma/client` package from your `package.json`.
+
+<Details><Summary><strong>Alternative:</strong> Clone the entire repo</Summary>
+
+Clone this repository
```

Comment:
Let's add a colon here for consistency:

```suggestion
Clone this repository:
```

Please also make sure the colon is added to all other examples where it's missing.